### ISSUE TITLE
Waterfox Classic help menu ▸ Waterfox help

### ISSUE THREAD

**Issue by grahamperrin** [BrowserWorks/waterfox#400]
From [Waterfox ▸ Help ▸ Submit feedback… · Issue #399 · MrAlex94/Waterfox](https://github.com/MrAlex94/Waterfox/issues/399): 

> … Focus instead on Help ▸ … Help …

In 56.0.3, choosing Waterfox help leads to Firefox help. 

As a **short-term** fix, maybe simply change the menu item to _Firefox help_. (Most people who find themselves using Waterfox will be aware of its Firefox origins.)

----

Longer term, have a _Waterfox help_ menu item leading to the support area of the Waterfox site. 

For example, from the current staging area: 

![2018-01-20 11 57 33 waterfox](https://user-images.githubusercontent.com/192271/35183167-793615c0-fdd9-11e7-8beb-43db0d25eca9.png)

**Comment by grahamperrin** [comment]
> … #400 is already fixed though, you may want to close this one.

@Peacock365 fixed in a release? Or in the PR above?

**Comment by Peacock365** [comment]
The website part is fixed already, so there is that. The in-browser part is going to be fixed soon, I assume. The issue could be closed at this point, for it is a duplicate of the bug I opened and is already partly fixed.

**Comment by grahamperrin** [comment]
Thanks, this issue does include a screenshot from the staging site (as it was) but it's not for website. 

I'm building at the moment, I'll leave this issue open until I see the fix.

**Comment by grahamperrin** [comment]
#538

**Comment by Peacock365** [comment]
@grahamperrin 

A fix might arrive soon according to the comment of @Toromino in #548. This could also fix #399.

**Comment by Peacock365** [comment]
This can be closed once https://github.com/MrAlex94/Waterfox/pull/573 is merged in.

**Comment by grahamperrin** [comment]
Cross reference [Waterfox Classic 2019.10 menu item "Help -> Waterfox Help" goes to Mozilla's Firefox Support Web page, which isn't very helpful](https://www.reddit.com/r/waterfox/comments/dvcvjm/waterfox_classic_201910_menu_item_help_waterfox/)
